doc/README.md

```diff
@@ -1,5 +1,18 @@
-![MongooseIM](MongooseIM_logo.png "MongooseIM's logo")
+# MongooseIM platform
 
+<img align="left" src="MongooseIM_logo.png" alt="MongooseIM platform's logo" />
+
+MongooseIM is Erlang Solutions' robust and efficient XMPP platform aimed at large installations. Specifically designed for enterprise purposes, it is fault-tolerant, can utilize resources of multiple clustered machines and easily scale in need of more capacity (by just adding a box/VM). MongooseIM can accept client sessions over vanilla XMPP, Websockets, HTTP long-polling (a.k.a. BOSH), and a REST API.
+
+The MongooseIM platform comes with server-side components and client libraries. We provide a test suite and a monitoring server. We recommand third-party, open source client libraries for XMPP and REST API.
+
+Its home on GitHub is at http://github.com/esl/MongooseIM.
```

Comment:
The URL is not rendered as a link on readthedocs.io: http://mongooseim.readthedocs.io/en/readme-streamline/